test/robot/MRkeywords.resource

```diff
@@ -6,14 +6,15 @@ Library    ModelRegistry.py
 
 *** Variables ***
 ${MR_HOST}     %{TEST_MR_HOST=localhost}
+${PORT}        %{TEST_MR_PORT=8080}
```

Comment:
just for consistency, what do you think of prefixing with `MR_` like for the other variable?

```suggestion
${MR_PORT}        %{TEST_MR_PORT=8080}
```

and related needed modifications in the below of the script?

Reply:
@tarilabs Thanks for suggestion, good catch 👍  Updated.